.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
Is there any value to Dependabot in a static site? My primary experience with it has been that it generates a lot of pointless noise and nearly zero useful PRs.

Reply:
Looking at the repo from inkydragon that seems to be exactly the case.
![image](https://user-images.githubusercontent.com/4618135/161397203-0b9ae9c2-3add-44fd-8fa2-06aa41b75202.png)

Reply:
I've deleted `dependabot.yml`.